components/dashboard-modules/EnergyRecommendation.tsx

```diff
@@ -0,0 +1,75 @@
+import Image from 'next/image';
+import { FC } from 'react';
+import { useSiteData } from '~/lib/hooks';
+import { getCurrentTimeForecast } from '~/lib/utils';
+import content from '../../content/power-card-content.json';
+import NumberDisplay from './NumberDisplay';
+
+/**
+ * Determines the appliance with the greatest energy required that is less than or equal to the current output
+ * @param currentOutput the current output in kW based on the getCurrentTimeForecast
+ * @returns the index of the appliance with the greatest enrgy requiement less than or equal to the current ouput
+ * or -1 if an index could not be found
+ */
+
+const getBestRecommendationIndex = (currentOutput: number) => {
+  if (currentOutput != undefined) {
+    let maxIndex = null;
+    let maxKW = 0;
+
+    content.appliances.forEach((appliance, i) => {
+      if (
+        Number(appliance.kW) > maxKW &&
+        Number(appliance.kW) <= currentOutput
+      ) {
+        maxIndex = i;
+        maxKW = Number(appliance.kW);
+      }
+    });
+
+    return maxIndex;
+  }
+  return null;
+};
+
+const EnergyRecommendation: FC<{ siteUUID: string }> = ({ siteUUID }) => {
+  const { forecastData, isLoading } = useSiteData(siteUUID);
+  const currentOutput = forecastData
+    ? getCurrentTimeForecast(forecastData.forecast_values)
+    : undefined;
+
+  // const recommendationIdx = currentOutput
+  //   ? getBestRecommendationIndex(currentOutput)
+  //   : null;
+
+  const recommendationIdx = getBestRecommendationIndex(5);
```

Comment:
Is this supposed to be changed to this?
```suggestion
  const recommendationIdx = getBestRecommendationIndex(currentOutput);
```